prep-html-syntax/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+
+<html>
+
```

Comment:
Hi Gregg, be sure to include a `<head></head>` containing a `<title></title>` before your body too.

Reply:
Ah okay, will do right now

Reply:
All set.